### DISCUSSION TITLE
elastcisearch and loopback4

### DISCUSSION THREAD

**Discussion by ScottYueqiang** [loopbackio/loopback-next#7176] [Q&A]
i get data from elastic, and the error happens:
Trace: [illegal_argument_exception] Fielddata is disabled on text fields by default. Set fielddata=true on [id] in order to load fielddata in memory by uninverting the inverted index. Note that this can however use significant memory. Alternatively use a keyword field instead.
    at D:\lb4\es\node_modules\loopback-connector-es\lib\esConnector.js:869:19
    at tryCallOne (D:\lb4\es\node_modules\promise\lib\core.js:37:12)
    at D:\lb4\es\node_modules\promise\lib\core.js:123:15
    at flush (D:\lb4\es\node_modules\asap\raw.js:50:29)
    at processTicksAndRejections (internal/process/task_queues.js:75:11)

below are my model, datasource config and controller function:
![image](https://user-images.githubusercontent.com/36142253/69307259-04347b80-0c65-11ea-8f9d-528fa8d4d796.png)
![image](https://user-images.githubusercontent.com/36142253/69307282-0f87a700-0c65-11ea-82f9-b6b0a6f1a7ca.png)
![image](https://user-images.githubusercontent.com/36142253/69307305-229a7700-0c65-11ea-9198-776601df0b98.png)

**Comment by emonddr** [comment]
@ScottYueqiang , perhaps contact the creator of the connector you are using? e.g. https://www.npmjs.com/package/loopback-connector-esv6 . The loopback team does not maintain the elastic search connector.
